### PR TITLE
Info-Popup for Search-Engine

### PR DESCRIPTION
**Pull request by danji90**
# How to
A "Help"-button has been added next to the WKP Search-Input, toggles a Popup with tips and examples for WKP searches

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [x] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] The title means something for a human being.
- [x] The title contains [WIP] if it's necessary.
- [x] Labels applied. if it's a release? a hotfix?
- [ ] Tests added.

### CODE REVIEW

File: src/lang/fr.json [hunk 1]
```diff
@@ -579,5 +579,19 @@
   "Zugtracker": "Train tracker",
   "zurück": "précédent",
   "Zurücksetzen": "Réinitialiser",
-  "Zusätzliche Informationen": "Informations complémentaires"
+  "Zusätzliche Informationen": "Informations complémentaires",
+  "oder": "ou",
```
Comment (danji90) [inline src/lang/fr.json:583]: @oterral Please check/correct the french translations

File: src/lang/fr.json [hunk 2]
```diff
@@ -588,5 +588,19 @@
...
+  "z.B.": "par ex.",
+  "search.info.stations.desc": "Saisie du nom officiel de la station",
+  "search.info.stations.example": "\"Bern Europaplatz\" pour une station de train ou \"Bern Europaplatz, Bahnhof\" pour une station de bus/tramway",
+  "search.info.municipalities.desc": "Saisie du nom de la commune",
+  "search.info.places.desc": "Recherche de noms de lieux, de cols, de montagnes, de cours d'eau, etc. à partir des cartes nationales",
+  "search.info.addresses.desc": "Saisie d'une adresse",
+  "search.info.operatingpoints.desc": "Recherche de points d'exploitation sur le réseau ou d'abréviations de points d'exploitation",
+  "search.info.lines.desc": "Saisie d'un numéro de ligne",
+  "search.info.lineskilometer.desc": "Recherche d'un kilomètre de ligne sur une ligne en saisissant le numéro de ligne et le kilomètre de ligne",
```
Comment (oterral) [inline src/lang/fr.json:601]:
```suggestion
  "search.info.lineskilometer.desc": "Recherche d'un kilomètre précis sur une ligne en saisissant le numéro de ligne puis le kilomètre souhaité",
```

File: src/lang/fr.json [hunk 3]
```diff
@@ -588,5 +588,19 @@
...
+  "oder": "ou",
+  "z.B.": "par ex.",
+  "search.info.stations.desc": "Saisie du nom officiel de la station",
+  "search.info.stations.example": "\"Bern Europaplatz\" pour une station de train ou \"Bern Europaplatz, Bahnhof\" pour une station de bus/tramway",
+  "search.info.municipalities.desc": "Saisie du nom de la commune",
+  "search.info.places.desc": "Recherche de noms de lieux, de cols, de montagnes, de cours d'eau, etc. à partir des cartes nationales",
+  "search.info.addresses.desc": "Saisie d'une adresse",
+  "search.info.operatingpoints.desc": "Recherche de points d'exploitation sur le réseau ou d'abréviations de points d'exploitation",
+  "search.info.lines.desc": "Saisie d'un numéro de ligne",
+  "search.info.lineskilometer.desc": "Recherche d'un kilomètre précis sur une ligne en saisissant le numéro de ligne puis le kilomètre souhaité",
```
Comment (oterral) [inline src/lang/fr.json:601]:
```suggestion
  "search.info.lineskilometer.desc": "Recherche d'un kilomètre précis d'une ligne en saisissant le numéro de ligne puis le kilomètre souhaité",
```

File: src/lang/fr.json [hunk 4]
```diff
@@ -588,5 +588,19 @@
...
+  "z.B.": "par ex.",
+  "search.info.stations.desc": "Saisie du nom officiel de la station",
+  "search.info.stations.example": "\"Bern Europaplatz\" pour une station de train ou \"Bern Europaplatz, Bahnhof\" pour une station de bus/tramway",
+  "search.info.municipalities.desc": "Saisie du nom de la commune",
+  "search.info.places.desc": "Recherche de noms de lieux, de cols, de montagnes, de cours d'eau, etc. à partir des cartes nationales",
+  "search.info.addresses.desc": "Saisie d'une adresse",
+  "search.info.operatingpoints.desc": "Recherche de points d'exploitation sur le réseau ou d'abréviations de points d'exploitation",
+  "search.info.lines.desc": "Saisie d'un numéro de ligne",
+  "search.info.lineskilometer.desc": "Recherche d'un kilomètre précis d'une ligne en saisissant le numéro de ligne puis le kilomètre souhaité",
+  "search.info.linessegment.desc": "Recherche d'un segment de ligne en saisissant le numéro de ligne et le kilomètre de - kilomètre à",
```
Comment (oterral) [inline src/lang/fr.json:602]:
```suggestion
  "search.info.linessegment.desc": "Recherche d'un segment précis d'une ligne en saisissant le numéro de ligne puis le kilomètre de - le kilomètre ]",
```

File: src/lang/fr.json [hunk 5]
```diff
@@ -588,5 +588,19 @@
...
+  "z.B.": "par ex.",
+  "search.info.stations.desc": "Saisie du nom officiel de la station",
+  "search.info.stations.example": "\"Bern Europaplatz\" pour une station de train ou \"Bern Europaplatz, Bahnhof\" pour une station de bus/tramway",
+  "search.info.municipalities.desc": "Saisie du nom de la commune",
+  "search.info.places.desc": "Recherche de noms de lieux, de cols, de montagnes, de cours d'eau, etc. à partir des cartes nationales",
+  "search.info.addresses.desc": "Saisie d'une adresse",
+  "search.info.operatingpoints.desc": "Recherche de points d'exploitation sur le réseau ou d'abréviations de points d'exploitation",
+  "search.info.lines.desc": "Saisie d'un numéro de ligne",
+  "search.info.lineskilometer.desc": "Recherche d'un kilomètre précis d'une ligne en saisissant le numéro de ligne puis le kilomètre souhaité",
+  "search.info.linessegment.desc": "Recherche d'un segment précis d'une ligne en saisissant le numéro de ligne puis le kilomètre de - le kilomètre ]",
```
Comment (oterral) [inline src/lang/fr.json:602]:
```suggestion
  "search.info.linessegment.desc": "Recherche d'un segment précis d'une ligne en saisissant le numéro de ligne puis le kilomètre de - le kilomètre à",
```

**Pull request merged**